test/tools/libtesteth/TestHelper.cpp

```diff
@@ -637,12 +637,6 @@ string prepareLLLCVersionString()
     return "Error getting LLLC Version";
 }
 
-#define STR(X) #X
```

Comment:
It should have been 
```
#define STR(X) #X
#define XSTR(X) STR(#X)
```
https://github.com/ethereum/evmc/blob/master/examples/example_vm.c#L180

Reply:
It was your code, can you fix it? 😉

Reply:
I'm ok with removing it.